models/db/engine.go

```diff
@@ -293,3 +293,25 @@ func GetMaxID(beanOrTableName interface{}) (maxID int64, err error) {
 	_, err = x.Select("MAX(id)").Table(beanOrTableName).Get(&maxID)
 	return
 }
+
+// QuoteString quotes the string for SQLs. In most cases, SQL should use the ORM builder.
+// This function could only be used in rare cases when there is no other choices.
+// It panics if the database is not supported to avoid corrupted data
+func QuoteString(ctx context.Context, s string) string {
```

Comment:
```suggestion
// QuoteString quotes the string for SQL. In most cases, SQL should use the ORM builder, not this one.
// This function is NOT recommended, it could ONLY be used in rare cases when there is no other choice.
// It panics if the database is not supported to avoid corrupted data
// At the moment it follows the behavior of convertString and convertStringSingleQuote in XORM, and is covered by tests.
func QuoteString(ctx context.Context, s string) string {
```